pkg/keyspace/keyspace_test.go

```diff
@@ -435,6 +435,43 @@ func (suite *keyspaceTestSuite) TestPatrolKeyspaceAssignmentInBatch() {
 	}
 }
 
+func (suite *keyspaceTestSuite) TestPatrolKeyspaceAssignmentWithRange() {
```

Comment:
How about also testing maxEtcdTxnOps? If we use a txn, will maxEtcdTxnOps affect only one kind of operation or all kinds of operations?